src/write/column_chunk.rs

```diff
@@ -26,11 +26,12 @@ use super::page::{write_page, PageWriteSpec};
 use super::statistics::reduce;
 use super::DynStreamingIterator;
 
-pub fn write_column_chunk<'a, W, E>(
+pub fn write_column_chunk<W, E>(
     writer: &mut W,
     mut offset: u64,
     descriptor: &ColumnDescriptor,
-    mut compressed_pages: DynStreamingIterator<'a, CompressedPage, E>,
+    mut compressed_pages: DynStreamingIterator<CompressedPage, E>,
+    #[cfg(feature = "bloom_filter")] bloom_filter_bitset: Option<&[u8]>,
```

Comment:
my minor concern with this approach (arguments depend on the feature set) is the support of Cargo docs to it - do you know if Cargo docs handles this correctly?

Reply:
fixed this